class-00/lab-b/starter-code/app.js

```diff
@@ -190,17 +163,18 @@ Student.prototype.scope = function() {
 };
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scope());
+console.log('joe scope', joe.scope());
 
-Student.prototype.scopeArrow = () => console.log(this);
+Student.prototype.scopeArrow = () => console.log('this', this);
 
 // TODO: Uncomment the following line of code to see the output in the browser console
-// console.log(joe.scopeArrow());
+console.log('arrow', joe.scopeArrow());
 
 // TODO: Write a COMMENT below to answer the following questions.
 // 1. What is "this" when joe.scope() is invoked?
-//
+//  this is undefined.
 // 2. What is "this" when joe.scopeArrow() is invoked?
-//
+//  this is an empty object

```

Comment:
The 'this' should not be an empty object. In fact, it probably is an object that has a comparatively extremely full of various parameters and values.